### PR TITLE
Add Input Variations

### PR DESCRIPTION
**Pull request by Johannafendrich**
Storybook preview:
https://my-homie-inputs-whlznyfaddcjq3.herokuapp.com/storybook/index.html?path=/story/inputs--mail-input
#20

### CODE REVIEW

File: client/src/stories/Input.stories.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import UserGroupIcon from '../assets/user-group.svg';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Inputs',
+};
+
+export const mailInput = () => (
+  <InputContainer>
+    <img src={EmailAtIcon} alt="email" />
```
Comment (florianGierlichs) [inline client/src/stories/Input.stories.js:19]: You should overthink your `alt` description.
The `alt` is used to give blind people a hint, what the image looks like. I don't know the EmailAtIcon or the PasswordIcon and I have no idea how they look like just on your `alt` description.

File: client/src/stories/Input.stories.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    <InputField type="email" placeholder="E-mail" onChange={action('write')} />
+  </InputContainer>
+);
+
+export const passwordInput = () => (
+  <InputContainer>
```
Comment (florianGierlichs) [inline client/src/stories/Input.stories.js:25]: You use multiple and the same Components to build your input.
I would highly suggest building one Input and just give it different props to have diversity.
for example:
```js
const Input = ({ src, alt, placeholder <ect.> }) => {
<InputContainer>
    <img src={src} alt={alt} />
    <InputField type="text" placeholder={placeholder} onChange={action('write')} />
  </InputContainer>
}
```

```js
const UserNameInput = () => {
<Input src={UserIcon} alt="name" placeholder="Name"  <ect.> />
}
```

Just a suggestion 😄

File: client/src/stories/Input.stories.js (hunk 1 above)
Comment (Johannafendrich) [reply client/src/stories/Input.stories.js:19]: You are right when I think that the placeholder text explains what kind of input field it is, here the old is actually double 😅 Thanks for the hint! :)

File: client/src/stories/Input.stories.js (hunk 2 above)
Comment (Johannafendrich) [reply client/src/stories/Input.stories.js]: Thanks for the helpful hint, it was not easy but I have now been able to combine it into one component as input field 👍🏼thanks!

**Comment by Johannafendrich** [comment]
> YOU'RE DOING IT JOHANNA! :D The only thing I would recommend at some point is consolidating those two styled components in `Input.js` so you only have one component that receives an icon and `type`, `placeholder` etc.
> well done! 🐜

Thank you, it has taken some time but I hope it's better now 😅✊🏻

**Pull request merged**